syntax = "proto3";

service Greeter {
rpc guardar_bloques (informacion_archivo) returns (asignacion);
}

message informacion_archivo {
    string nombre_archivo = 1;
    int32 bloques = 2;
}

message asignacion {
    string message = 1;
}